inlong-agent/agent-plugins/pom.xml

```diff
@@ -79,6 +84,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-connector-mysql-cdc</artifactId>
+            <version>${flink.connector.mysql.cdc.version}</version>
```

Comment:
it's better to avoid using the `flink-connector-mysql-cdc` dependency.

Reply:
resolve